orttraining/orttraining/training_api/include/onnxruntime_training_c_api.h

```diff
@@ -173,6 +168,18 @@ struct OrtTrainingApi {
   */
   ORT_API2_STATUS(SetLearningRate, _Inout_ OrtTrainingSession* sess, _In_ float learning_rate);
 
+  /** \brief Gets the cuttent learning rate for this training session.
+  *
+  * This function allows users to get the learning rate for the training session. The current
+  * learning rate is maintained by the training session
+  *
+  * \param[in] sess The training session on which the learning rate needs to be set.
+  *
+  * \snippet{doc} snippets.dox OrtStatus Return Value
+  *
+  */
+  ORT_API2_STATUS(GetLearningRate, _Inout_ OrtTrainingSession* sess, _Out_ float& learning_rate);
```

Comment:
Why is this function needed?

Reply:
For testing and logging that the user may need